src/redux/books/books.js

```diff
@@ -0,0 +1,28 @@
+const ADD = 'bookstore/books/addBook';
+const REMOVE = 'bookstore/books/removeBook';
+
+const initialState = {
+  books: [],
+};
```

Comment:
- [ Required ] Please consider setting the `initialState` to an array as specified in the project requirements, currently, the `initialState` is an object. You can do this by refactoring this code as shown below.
```JavaScript
 const initialState = []
```
## Project requirements
![Screenshot from 2023-02-03 00-09-08](https://user-images.githubusercontent.com/16392046/216450883-a99a9b30-d2fb-43f6-bc1a-4318ba094203.png)